src/Deposit.js

```diff
@@ -643,9 +667,8 @@ export default class Deposit {
     )
     const proofArgs = await this.constructFundingProof(
       tx,
-      parseInt(requiredConfirmations)
+      requiredConfirmations
```

Comment:
This worked because `parseInt` doesn't do anything when passed a number, but it was redundant.